packages/extension/src/components/molecules/NetworkIndicator/NetworkIndicator.tsx

```diff
@@ -79,8 +110,33 @@ const NetworkDisplay: FC<NetworkDisplayProps> = ({
 
 export const NetworkIndicator: FC = () => {
   const { client, network, switchNetwork } = useNetwork();
+  const [currentNetworkName, setCurrentNetworkName] = useState<string>(network as string);
   const [explanationOpen, setExplanationOpen] = useState(false);
   const [isLoading, setIsLoading] = useState(false);
+  const [handleAddNetwork, setHandleAddNetwork] = useState(false);
+  const [existingNetworks, setExistingNetworks] = useState<
+    Record<string, { name: string; server: string; description?: string }>
```

Comment:
The type: `name: string; server: string; description?: string` is used 3 times in the code, maybe it could be extracted and imported 3 times

Reply:
Done